File: IOT_Pet_Planter/src/main/java/com/Whitecape/PFE/models/Plant.java
package com.Whitecape.PFE.models;

import org.springframework.data.annotation.Id;
import org.springframework.data.mongodb.core.mapping.Document;

@Document(collection="Plant")
public class Plant {
	@Id
	private String id;
    private String name;
    private String type;
    private String description;
    public Plant() {
    
    }
    
	public Plant(String name, String type, String description) {
		this.name = name;
		this.type = type;
		this.description = description;
	}

	public String getId() {
		return id;
	}

	public void setId(String id) {
		this.id = id;
	}

	public String getName() {
		return name;
	}

	public void setName(String name) {
		this.name = name;
	}

	public String getType() {
		return type;
	}

	public void setType(String type) {
		this.type = type;
	}

	public String getDescription() {
		return description;
	}

	public void setDescription(String description) {
		this.description = description;
	}
	
    
    
}

File: ngPlanter/src/app/app/plant.ts
export class Plant {
    plantId: string;
    plantName: string;
    type: string;
    description: string;
  
    constructor(plantId: string, plantName: string, type: string, description: string) {
      this.plantId = plantId;
      this.plantName = plantName;
      this.type = type;
      this.description = description;
    }
  }
  
